app/main/views/marketplace.py

```diff
@@ -90,21 +92,40 @@ def get_brief_by_id(framework_slug, brief_id):
 
 @main.route('/<framework_slug>/opportunities')
 def list_opportunities(framework_slug):
-    page = request.args.get('page', default=1, type=int)
     framework = data_api_client.get_framework(framework_slug)['frameworks']
-
     if not framework:
         abort(404, "No framework {}".format(framework_slug))
 
-    api_result = data_api_client.find_briefs(status='live,closed', framework=framework_slug,
-                                             page=page, human=True)
+    # disabling csrf protection as this should only ever be a GET request
+    form = BriefSearchForm(request.args, framework=framework, data_api_client=data_api_client, csrf_enabled=False)
```

Comment:
I think csrf protection is only enabled in the global config for POST requests to the app, so it might not be necessary to disable it here.